client/src/app/services/authentication.service.ts

```diff
@@ -18,19 +19,25 @@ export class AuthenticationService {
     return this.currentUserSubject.value;
   }
 
-  public login(username: string, password: string):boolean{
-    const hardCodedUser:User = {
-      id:1,
-      userName:'admin',
-      password:'admin123',
-      token:'token'
+  public login(username: string, password: string): Observable<any> {
+    const user: User = {
+      id: 1,                     //do I need that?
```

Comment:
I dont think so. What are you going to do with userId ?

Reply:
Deleted id field